app/models/visualization/overlays.rb

```diff
@@ -8,6 +8,8 @@ def initialize(visualization)
       end
 
       def create_default_overlays
+        prof_attrs = @visualization.user && @visualization.user.profile_attributes
+        puts "create_default_overlays - prof_attrs: #{prof_attrs}"
```

Comment:
Remove trace or place in log if useful

Reply:
Will remove - didn't mean to include this.

Reply:
Rubocop will aid in preventing things like this too! Please check it out